e2e-app/src/app/datepicker/autoclose/datepicker-autoclose.e2e-spec.ts

```diff
@@ -147,6 +147,23 @@ describe('Datepicker Autoclose', () => {
         await clickOnDay(DATE_OUTSIDE_AFTER);
         await expectDatepickerToBeClosed(`Datepicker should be closed on outside day click`);
       });
+      it(`should change autoClose setting dynamically`, async() => {
+        // initially set autoClose to true
+        await selectAutoClose('true');
```

Comment:
Can't really see the point in setting `autoClose` to `true` initially, closing and resetting it to `false`

Reply:
Calling selectAutoClose constitutes an outside click, which interferes with the test. I will change to testing with 'inside' setting.